pkg/crd/kubermatic/v1/cluster.go

```diff
@@ -190,6 +193,8 @@ var AllClusterConditionTypes = []ClusterConditionType{
 	ClusterConditionClusterControllerReconcilingSuccess,
 	ClusterConditionAddonControllerReconcilingSuccess,
 	ClusterConditionBackupControllerReconcilingSuccess,
+	ClusterConditionEtcdRestoreControllerReconcilingSuccess,
+	ClusterConditionBackupScheduleControllerReconcilingSuccess,
```

Comment:
:-O OK

Reply:
Looked at it again -- the new backup controller uses the old backup controller's condition (ClusterConditionBackupControllerReconcilingSuccess) [here](https://github.com/multi-io/kubermatic/blob/etcd-backups-controller/pkg/controller/seed-controller-manager/etcdbackup/etcd_backup_controller.go#L184). So it sets it to true the first time a backup was reconciled successfully for the cluster. I.e. it won't be set to true until a backupconfig is created for the cluster -- which it will be once I've added the code that adds a default one for every cluster, but it still feels wrong to rely on that, right? So it comes back to what I wrote in the PR description:

```
what ClusterReconcileWrapper() is doing here probably makes no sense for BackupController:

SetClusterCondition(cluster, conditionType, reconcilingStatus, "", "")

(because etcdbackup resources are supposed to be created continuously, there's not really one time when the controller's reconciliation is "finished" for this cluster. Maybe make this activity optional in ClusterReconcileWrapper()?
```

So I've now removed the unused conditions and also extended `ClusterReconcileWrapper()` a little bit so the "set a cluster condition" part is optional, and I've adapted the new backup controller so it no longer sets a condition. If some other solution is preferred, I can change it back easily.

Reply:
Now also discussed this in "Special notes for your reviewer:" above.